src/main/java/org/commcare/formplayer/application/FormSubmissionController.java

```diff
@@ -85,6 +104,9 @@ public class FormSubmissionController extends AbstractBaseController {
 
     private final Log log = LogFactory.getLog(FormSubmissionController.class);
 
+    @Value("${max.attachments.per.form}")
+    private Integer maxAttachmentsPerForm;
```

Comment:
@snopoke I am struggling with an issue with this property not bein initialised in the `MediaCaptureTest` . I tried adding annotations like `@TestPropertySource(properties = ["max.attachments.per.form=1"])` or `@TestPropertySource("classpath:application.properties")` to `MediaCaptureTest` , though this property remains `null`. Would you be able to have a look at it whenever you have some space.

Reply:
`@TestPropertySource("classpath:application.properties")` is what I've used before. You might need to add ` @ContextConfiguration` to the class as well.

Reply:
thanks I did try it but it doesn't seem to work. My suspicion is somehow `@WebMvcTest` is interferring with setting the property but I am out of ideas on how to debug it.